src/request.rs

```diff
@@ -9,6 +10,11 @@ use async_std::task::{Context, Poll};
 use std::pin::Pin;
 use std::{str::FromStr, sync::Arc};
 
+use crate::error::{Error, ResultExt, StringError};
+use crate::middleware::cookies::CookieData;
+
+const MIDDLEWARE_MISSING_MSG: &str = "`CookiesMiddleware` has not been enabled";
```

Comment:
We should enable the cookies middleware by default; removing the need for this string and the corresponding logic.